javacutil/src/main/java/org/checkerframework/javacutil/AbstractTypeProcessor.java

```diff
@@ -139,11 +139,20 @@ public void typeProcessingStart() {}
      * <p>Subclasses may override this method to do any aggregate analysis (e.g. generate report,
      * persistence) or resource deallocation.
      *
-     * <p>If an error (a Java error or a processor error) is reported, this method is not guaranteed
-     * to be invoked.
+     * @param hasError true if compilation issued an error, either from the Java compiler or from a
```

Comment:
You don't want this change.
But do change to say that the method is guaranteed to be called.
The Javadoc should probably contain a hint about getCompilerLog.nerrors.